blog/2022-08-15-connect-pit-to-stryker-dashboard.md

```diff
@@ -0,0 +1,78 @@
+---
+slug: connect-pit-to-stryker-dashboard
+title: 'Connect PIT to Stryker Dashboard'
+author: Maarten Mulders
+author_title: Stryker Team
+author_url: https://maarten.mulders.it/
```

Comment:
What do you expect here? Is it always "Stryker Team", or could it also be something else, e.g. "External Contributor", "Some Rando" or "Java Idiot"?

Reply:
I like Java Idiot 😉. I think in most cases we've used Stryker Team/Stryker.NET team/Stryker4s team. You're still sort of part of the team 🙂 so feel free to put there what you want

Reply:
I'll stick with "Stryker Team", in that case :-)